src/index.js

```diff
@@ -0,0 +1,11 @@
+import './style.css';
+import myTaskList from './modules/TaskList.js';
+import populateTasks from './modules/populateTasks.js';
+
+window.onload = () => {
+  const storedTasks = JSON.parse(window.localStorage.getItem('storedTasks'));
+  if (storedTasks) {
+    myTaskList.tasks = storedTasks;
+  }
```

Comment:
I suggest you can optimize this decision logic with the _Short-circuit Evaluation _ method to return the myTaskList.tasks = storedTasks;. That will keep the code simple. 😃